src/com/softwaret/kpdf/controller/login/LoginControllerImpl.kt

```diff
@@ -7,32 +7,41 @@ import com.softwaret.kpdf.response.Response
 import com.softwaret.kpdf.response.Unauthorized
 import com.softwaret.kpdf.response.error.ErrorResponseBody
 import com.softwaret.kpdf.response.success.LoginResponseBody
+import com.softwaret.kpdf.service.validation.input.InputValidator
+import com.softwaret.kpdf.util.extension.areAllNull
+import com.softwaret.kpdf.util.extension.get
 import com.softwaret.kpdf.util.parameters.BodyParameter.LOGIN
 import com.softwaret.kpdf.util.parameters.BodyParameter.PASSWORD
-import com.softwaret.kpdf.util.extension.get
-import com.softwaret.kpdf.util.extension.isAnyNotNull
 import io.ktor.http.Parameters
 
 class LoginControllerImpl(
-    private val loginInteractor: LoginInteractor
+    private val interactor: LoginInteractor,
+    private val inputValidator: InputValidator
 ) : BaseController(), LoginController {
 
     override suspend fun login(parameters: Parameters) =
```

Comment:
Taka mnie myśl naszła, może zamiast `Parameters` zrobić tutaj `vararg parameters` czy coś podobnego aby pozbyć się platformowego `import io.ktor.http.Parameters`

Reply:
Wyłączyłem to tak żeby controllery tylko przyjmowały gotowe parametry. Uno łatwiej testować i duo mniej w nich syfu.